ci/cudf_pandas_scripts/run_tests.sh

```diff
@@ -50,3 +55,10 @@ else
 fi
 
 python -m pytest -p cudf.pandas ./python/cudf/cudf_pandas_tests/
+
+python -m pytest -p cudf.pandas \
```

Comment:
It seems like the only difference between these two `pytest` invocations is the codecov flags.

Can they be combined? Otherwise won't the tests run twice?

We use a single `pytest` invocation in other scripts like https://github.com/rapidsai/cudf/blob/branch-24.02/ci/test_python_cudf.sh.